### ISSUE TITLE
SkypeWeb duplicates already received incoming messages

### ISSUE THREAD

**Issue by rand256** [miranda-ng/miranda-ng#1777]
It seems that SkypeWeb duplicates recent already received incoming messages on connect, but only if there are new unread offline messages.

I.e., if you have an online conversation with someone then everything's ok. But if you reconnect after a while when there're some unread offline messages then the incoming messages in that conversation (for last ~month at least) get duplicated in the log and history.

It got duplicated only once, so in history there's only one "genuine" incoming message and a one duplicate, it won't create duplicates of duplicates endlessly.

This happens for a few months at least, although maybe I hadn't had frequent offline messages before that.

**Comment by rand256** [comment]
I got offline message again today, and the whole history with that contact now has its incoming messages duplicated for at least a month, so the history is flooded by duplicates and awkwardly to read.

I'd suggest to add some auto-cleaning procedure to get such duplicates deleted when (if) this bug is fixed. All the duplicated messages has exactly the same datetime marks and texts.
